include/main.h

```diff
@@ -0,0 +1,17 @@
+// include/main.h
+#ifndef MAIN_H
+#define MAIN_H
+#include <array>
+#include <string_view>
+
```

Comment:
no clue how the main.h should look like, or even if we need one

Reply:
we can keep it for now and remove in the future if we find it's redundant